app/models/manageiq/providers/google/cloud_manager/event_catcher.rb

```diff
@@ -1,3 +1,9 @@
 class ManageIQ::Providers::Google::CloudManager::EventCatcher < ManageIQ::Providers::BaseManager::EventCatcher
   require_nested :Runner
+
+  def self.all_valid_ems_in_zone
+    # Only valid to start an EventCatcher if the Pub/Sub service is enabled
+    # on the project
+    super.select { |ems| ems.capabilities["pubsub"] }
```

Comment:
NOTE I think this would read well as `ems.supports?(:events)` however we don't have a top-level `:events` feature on `ExtManagementSystem` but I'm going to add one since we already have `supports?(:metrics)`